[release/7.0][wasm] Add CI job for AOT+SIMD to `runtime-wasm` for PRs

radical opened the pull request:
- Adds a job to `runtime-wasm` that runs all the AOT tests with SIMD enabled
- Also, add two `System.Runtime.Intrinsics` runs on helix with, and without simd
   - And run this with smoke tests too
- And add a `runtime-wasm-features` pipeline yml with the AOT+SIMD job. This isn't usable right now because we can't add new pipelines.

radical commented:
/azp run runtime-wasm

radical commented:
/azp run runtime-wasm

radical commented:
/azp run runtime-wasm

radical commented:
/azp run runtime-wasm

radical commented:
/azp run runtime-wasm

radical commented:
/azp run runtime-wasm

radical commented:
/azp run runtime-wasm

radical commented:
/azp run runtime-wasm

radical commented:
/azp run runtime-wasm

radical commented:
/azp run runtime-wasm

radical committed:
[wasm] Add support for running `System.Runtime.Intrinsics` tests with

.. SIMD enabled, as part of smoke tests.

radical committed:
[wasm] Add a new `runtime-wasm-features` pipeline

.. which has only a SIMD_AOT job for now.

radical committed:
[wasm] Add SIMD_AOT job to runtime-wasm for now, as we can't add new pipelines

radical commented:
/azp run runtime-wasm

radical committed:
Fix browser-simd run

radical commented:
/azp run runtime

radical commented:
/azp run runtime-wasm

radical commented:
/azp run runtime-wasm

carlossanlop commented:
Since this is an infra/test only change, it's tell mode, so I only need a sign-off from @lewing || @steveisok || @akoeplinger || @radekdoulik .

And of course, a confirmation from @radical that this is ready to merge (in case you need to push more changes and/or execute more `runtime-wasm` jobs).

radical commented:
Failures are relevant, I'll fix.

radical committed:
[wasm] Mark simd job as unstable for now

Issues:
https://github.com/dotnet/runtime/issues/75044
and https://github.com/dotnet/runtime/issues/75098

radical commented:
/azp run runtime-wasm

radical commented:
The PR is ready. There are simd test failures which will be fixed by other follow up PRs.

carlossanlop commented:
Thanks @radical .
Infra/test change only.
Signed off.
CI is green.
Ready to merge. :shipit:

radical commented:
/backport to main
